packages/data-stores/src/verticals/resolvers.ts

```diff
@@ -20,7 +20,7 @@ import { receiveVerticals } from './actions';
  * API response.
  */
 export function* getVerticals() {
-	const url = 'https://public-api.wordpress.com/wpcom/v2/verticals';
+	const url = '/calypso/page-templates/vertical-list.json';
```

Comment:
There's no reason to use a resolver for a static value. The selector can return the value.

Reply:
😕 This isn't a static value… we're adding a file to calypso and then using a network request to get a JSON response from it via a static URL?

This doesn't seem quite right. If we'd like to provide the contents of a local JSON file, let's require the JSON file and return it.

What's being done here right now seems off.

Reply:
@sirreal This PR is just for a demo to see how the format of the new verticals works in onboarding. The idea is to serve the content from an API.

Reply:
Sounds good. Let's avoid temporarily publishing this at a public URL, just require and return it.